package.json

```diff
@@ -25,6 +25,7 @@
     "lint": "xo src/**/*.js test/**/*.js",
```

Comment:
I added task to the project: https://github.com/Syncano/syncano-server-js/projects/1#card-2807188